### DISCUSSION TITLE
LISTEN_INADDR_ANY 设置为0 不起作用

### DISCUSSION THREAD

**Discussion by pasteman** [DIYgod/RSSHub#12026] [Q&A] [answered]
LISTEN_INADDR_ANY 设置为0 仍然可以通过”IP+端口号“的方式访问，请问怎么样可以禁止”IP+端口号“的访问方式？

**Comment by DIYgod** [comment, marked as answer]
这个参数是是否允许公网连接，不是是否允许 IP 连接

什么场景需要禁止 IP 访问

**Comment by pasteman** [comment]
尴尬，我以为这个参数是关闭 ip+端口 的方式访问。原本打算用nginx反向代理，通过域名访问，然后关掉 ip+端口号的访问方式
